riscv/interactive.cc

```diff
@@ -732,7 +732,8 @@ void sim_t::interactive_until(const std::string& cmd, const std::vector<std::str
     throw trap_interactive();
 
   // mask bits above max_xlen
-  int max_xlen = procs[strtol(args[1].c_str(),NULL,10)]->get_isa().get_max_xlen();
+  bool until_mem_paddr = args[0] == "mem" && args.size() == 3;
+  int max_xlen = until_mem_paddr? procs[0]->get_isa().get_max_xlen() : procs[strtol(args[1].c_str(),NULL,10)]->get_isa().get_max_xlen();
```

Comment:
Make a separate variable for the proc number, instead of repeating the entire expression. Something like:
```
size_t procnum = until_mem_paddr ? 0 : strtol(args[1].c_str(), NULL, 10);
int max_xlen = procs[procnum]->get_isa().get_max_xlen();
```

Reply:
This has been fixed. New review is needed.